src/ontology/cl-edit.owl

```diff
@@ -2659,6 +2658,12 @@ AnnotationAssertion(rdfs:label rdfs:seeAlso "see also"^^xsd:string)
 #   Object Properties
 ############################
 
+# Object Property: obo:RO_0013001 (has synaptic IO in)
```

Comment:
Is 'IO' an acronym? I'm not sure what the requirements are for labels in RO, but typically labels ought not contain acronyms.

Reply:
I've copied it directly from http://purl.obolibrary.org/obo/RO_0013001
have now added synonym to clarify it (similar to above)

Reply:
Ah, okay. If it's okay with RO, it's okay with me.